tests/Rule/LatteTemplatesRule/PresenterWithoutModule/Fixtures/ResolvePresenter.php

```diff
@@ -94,4 +94,10 @@ public function actionSendTemplateUnresolvable(): void
         $this->template->send = 'send';
         $this->sendTemplate(new DefaultTemplate(new Engine()));
     }
+
+    public function dynamicClassName()
+    {
+        $class = uniqid();
```

Comment:
What about some string representing existing class name? It should work.

Reply:
I think we could use value resolver if it is expression

Reply:
You have scope here, isn't it better?

Reply:
Value resolver uses scope internally in cases where it is useful

Reply:
👍